actors/builtin/miner/miner_actor.go

```diff
@@ -1761,55 +1812,71 @@ func enrollCronEvent(rt Runtime, eventEpoch abi.ChainEpoch, callbackPayload *Cro
 	builtin.RequireSuccess(rt, code, "failed to enroll cron event")
 }
 
-func requestUpdateSectorPower(rt Runtime, sectorSize abi.SectorSize, sectorsAdded, sectorsRemoved []*SectorOnChainInfo) (rawDelta, qaDelta big.Int) {
+func requestUpdatePowerForSectors(rt Runtime, sectorSize abi.SectorSize, sectorsAdded, sectorsRemoved []*SectorOnChainInfo) (delta PowerPair) {
```

Comment:
This (called only from new sector assignment) breaks the pattern now used elsewhere of having the power returned from the partition-mutating method (which computes it for the expiration queue).

Reply:
Fixed.